src/index.tsx

```diff
@@ -52,6 +52,8 @@ export interface ReactDiffViewerProps {
   hideRightLineNumbers?: boolean;
   // Show only diff between the two values.
   showDiffOnly?: boolean;
+  // Return vertical view's nodes as an array of JSX Elements with no titles.
+  returnDiffNodesArray?: boolean;
```

Comment:
`returnArray` is implementation details, rather better have a more descriptive way to describe this prop